arlas-core/src/main/java/io/arlas/server/model/CollectionReferenceParameters.java

```diff
@@ -89,27 +91,34 @@
     public Filter filter = null;
 
     @JsonIgnore
-    private transient GeoTypeEnum geometryType;
-
-    @JsonIgnore
-    private transient GeoTypeEnum centroidType;
+    private Map<String, GeoTypeEnum> geoTypes = new ConcurrentHashMap<>();
 
     public CollectionReferenceParameters() {
     }
 
+    public void setGeoType(String path, GeoTypeEnum type) {
```

Comment:
is it possible to keep only `setGeoType` and `getGeoType` methods ? `setGeometryType` can be misleading as a method name : it doesn't say necessarily we set the geometry_path type.

Also if it is possible, maybe we can rename `setGeoType` to `setGeometryType`

Reply:
Done!